shared/src/main/java/net/skinsrestorer/shared/interfaces/ISRPlugin.java

```diff
@@ -57,18 +58,23 @@ public interface ISRPlugin {
 
     ISkinCommand getSkinCommand();
 
+    LocaleManager<ISRForeign> getLocaleManager();
+
     @SuppressWarnings({"deprecation"})
     default void prepareACF(CommandManager<?, ?, ?, ?, ?, ?> manager, SRLogger srLogger) {
         // optional: enable unstable api to use help
         manager.enableUnstableAPI("help");
+        LocaleManager<ISRForeign> localeManager = getLocaleManager();
+
+        SkinsRestorerAPIShared api = SkinsRestorerAPIShared.getApi();
 
         CommandReplacements.permissions.forEach((k, v) -> manager.getCommandReplacements().addReplacement(k, v.call()));
-        CommandReplacements.descriptions.forEach((k, v) -> manager.getCommandReplacements().addReplacement(k, v.call()));
-        CommandReplacements.syntax.forEach((k, v) -> manager.getCommandReplacements().addReplacement(k, v.call()));
+        CommandReplacements.descriptions.forEach((k, v) -> manager.getCommandReplacements().addReplacement(k, localeManager.getMessage(api.getDefaultForeign(), v.call().getKey())));
+        CommandReplacements.syntax.forEach((k, v) -> manager.getCommandReplacements().addReplacement(k, localeManager.getMessage(api.getDefaultForeign(), v.call().getKey())));
         CommandReplacements.completions.forEach((k, v) -> manager.getCommandCompletions().registerAsyncCompletion(k, c ->
-                Arrays.asList(v.call().split(", "))));
+                Arrays.asList(localeManager.getMessage(api.getDefaultForeign(), v.call().getKey()).split(", "))));
 
-        CommandPropertiesManager.load(manager, getDataFolderPath(), getResource("command-messages.properties"), srLogger);
+        CommandPropertiesManager.load(manager, getDataFolderPath(), getResource("command.properties"), srLogger);
```

Comment:
is this not missing some code to locate translated property file?

Reply:
Built-in messages from acf are not translated currently.